packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/prometheus-exporter/metrics.ts

```diff
@@ -0,0 +1,7 @@
+import { Gauge } from "prom-client";
+
+export const totalTxCount = new Gauge({
+  name: "totalTxCount",
```

Comment:
@jagpreetsinghsasan While I was testing this, typing it to the Prometheus expression console, I realized that once we have multiple metrics, the naming here would become ambiguous. Any chance that you could update the metrics name to uniquely identify that 1) it's a Cactus metric and 2) within Cactus it's specific to fabric?
Something like this `cactus_fabric_total_tx_count` could work (and this would have the added benefit of following the (seeming) convention in Prometheus to have underscores in the names, not camelCase

Reply:
fixed